src/main/controllers/helpers/PageContentHelpers.ts

```diff
@@ -85,3 +86,11 @@ export const getApplicationRespondByDate = (
     return dateString;
   }
 };
+
+export function formatCaseDateDDMMYYYY(caseDate: CaseDate): string {
```

Comment:
sorry, any chance you could add some error handling here? just in case date formats change silently and we're screwed?
Alternatively use luxon and it should throw an error automatically. It's a really nice lib, and i think it's probably more best practice than manual string parsing.